src/curp-command.proto

```diff
@@ -103,34 +153,24 @@ message ShutdownResponse {
 
 message ProposeConfChangeRequest {
     enum ConfChangeType {
-        Add = 0;
-        AddLearner = 1;
-        Remove = 2;
-        Update = 3;
-        Promote = 4;
+      Add = 0;
+      AddLearner = 1;
+      Remove = 2;
+      Update = 3;
+      Promote = 4;
     }
     message ConfChange {
-        ConfChangeType change_type = 1;
-        uint64 node_id = 2;
-        repeated string address = 3;
+      ConfChangeType change_type = 1;
+      uint64 node_id = 2;
+      repeated string address = 3;
     }
     ProposeId propose_id = 1;
     repeated ConfChange changes = 2;
     uint64 cluster_version = 3;
 }
 
 message ProposeConfChangeResponse {
-    oneof error {
-        google.protobuf.Empty InvalidConfig = 1;
-        google.protobuf.Empty NodeNotExists = 2;
-        google.protobuf.Empty NodeAlreadyExists = 3;
-        google.protobuf.Empty LearnerNotCatchUp = 4;
-        ProposeError propose = 5;
-        string other = 6;
-    }
-    repeated Member members = 7;
-    optional uint64 leader_id = 8;
-    uint64 term = 9;
+    repeated Member members = 1; 
```

Comment:
Same as above

Reply:
RedirectError contains the leader id and term, which should not be cared about here.